src/.prototype/syntaxElements/primitiveElements.ts

```diff
@@ -113,6 +113,30 @@ export class TInt extends PrimitiveElement<number> {
     static mod(operand_1: TInt, operand_2: TInt) {
         return new TInt(operand_1.value % operand_2.value);
     }
+        /** Binary == operator. */
+    static equals(operand_1: TInt, operand_2: TInt) {
+        return new TBoolean(operand_1.value == operand_2.value);
+    }
+
+    /** Binary > operator. */
+    static greaterThan(operand_1: TInt, operand_2: TInt) {
+        return new TBoolean(operand_1.value > operand_2.value);
+    }
+
+    /** Binary < operator. */
+    static lessThan(operand_1: TInt, operand_2: TInt) {
+        return new TBoolean(operand_1.value < operand_2.value);
+    }
+
+    /** Binary ++ operator. */
+    static increment(operand_1: TInt) {
+        return new TInt(operand_1.value + 1)
+    }
+
+    /** Binary -- operator. */
+    static decrement(operand_1: TInt) {
+        return new TInt(operand_1.value - 1)
+    }
```

Comment:
Unary operators.

Unary operators wouldn't return a new primitive object, instead they'll update the current object.

e.g.
```typescript
increment() {
    (this.value as number)++;
}
```

Reply:
Thanks used this syntax.
```
increment() {
    ((this.value +1) as number);
}
```

reason:- [link](https://dev.to/theodesp/comment/799h)